tools/mypy-onnx.py

```diff
@@ -13,7 +13,6 @@ def main():  # type: () -> None
         os.chdir(root_folder)
         # Use --no-site-packages to prevent mypy catching other typecheck errors which are not related to ONNX itself
         subprocess.check_call(["mypy", ".", "--no-site-packages"])
-        subprocess.check_call(["mypy", "--py2", ".", "--no-site-packages"])
```

Comment:
For others' reference: https://github.com/python/mypy/issues/10994#issuecomment-901621815 Remove python 2 typecheck since it cannot use dynamic dictionary... It should be OK since ONNX does not support Python 2 for a while